engine/src/main/java/org/hibernate/search/analyzer/spi/AnalyzerDefiner.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.analyzer.spi;
+
+import org.apache.lucene.analysis.Analyzer;
+
+/**
+ * Applies to index managers that are bound to backends supporting built-in analyzer definitions.
+ * <p>
+ * For example, Elasticsearch comes with whitespace and languages analyzers without definition defined in
+ * the classpath.
+ *
+ * @author Davide D'Alto
+ */
+public interface AnalyzerDefiner {
+
+	/**
+	 * Checks if the index manager can managed additional analyzer
+	 *
+	 * @return true if it supports additional analyzers without the need of an analyzer defintion
+	 */
+	boolean supportsAdditionalAnalyzer();
```

Comment:
Right, make sense.
it's a left over from one of my experiments, when I initially wanted to pass the name of the analyzer to check the existence,  I will remove it since it seems it cannot be done at the moment